<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <widget class="QWidget" name="tabWidgetPage1">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <attribute name="icon">
         <iconset>
          <normaloff>icons/kubbetussahra_128x128x32.png</normaloff>icons/kubbetussahra_128x128x32.png</iconset>
        </attribute>
        <attribute name="title">
         <string>إعدادات المدينة</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QListWidget" name="listCountries"/>
           </item>
           <item>
            <widget class="QListWidget" name="listCities"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="title">
              <string>التوقيت</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <spacer name="horizontalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QRadioButton" name="rbWinter">
                  <property name="text">
                   <string>شتوي</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QRadioButton" name="rbSummer">
                  <property name="text">
                   <string>صيفي</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="title">
              <string>بيانات المدينة</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLineEdit" name="txtLongitude"/>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>خط الطول</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLineEdit" name="txtLatitude"/>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>خط العرض</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLineEdit" name="txtTimeZone"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>فرق التوقيت عن جرينتش</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="cboxMazhab">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <item>
              <property name="text">
               <string>الشافعي,المالكي,الحنبلي</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>الحنفي</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>المذهب</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="cboxCalendar">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <item>
              <property name="text">
               <string>جامعة أم القري</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>الهيئة  المصرية للمساحة</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>جامعة العلوم الأسلامية بكراتشي</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>الأتحاد الأسلامي بأمريكا الشمالية</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>رابطة العالم الأسلامي</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>التقويم</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabWidgetPage2">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <attribute name="icon">
         <iconset>
          <normaloff>icons/preferences-desktop-sound.png</normaloff>icons/preferences-desktop-sound.png</iconset>
        </attribute>
        <attribute name="title">
         <string>إعدادات الأذان</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>صوت الأذان</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QToolButton" name="btnStop">
             <property name="minimumSize">
              <size>
               <width>33</width>
               <height>33</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>icons/media-playback-pause.png</normaloff>icons/media-playback-pause.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnPlay">
             <property name="minimumSize">
              <size>
               <width>33</width>
               <height>33</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>icons/media-playback-start.png</normaloff>icons/media-playback-start.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cboxAzanSound">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <item>
              <property name="text">
               <string>مصر- ناصر القطامي</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>المسجد الحرام</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>المسجد النبوي</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>مصر - عبدالباسط عبد الصمد</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>344</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>إغلاق</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveSettings">
       <property name="text">
        <string>حفظ  </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
